test/tests/restart/restart_subapp_not_master/two_step_solve_sub_restart.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  file = two_step_solve_master_cp/0002_mesh.cpr
+[]
+
+[Problem]
+  restart_file_base = two_step_solve_master_cp/LATEST
+  allow_sub_app_restart = true
+[]
+
+[Functions]
+  [./exact_fn]
+    type = ParsedFunction
+    value = t*t*(x*x+y*y)
+  [../]
+
+  [./forcing_fn]
+    type = ParsedFunction
+    value = 2*t*(x*x+y*y)-4*t*t
+  [../]
+[]
+
+[Variables]
+  [./u]
+    family = LAGRANGE
+    order = SECOND
+  [../]
+[]
+
+# Initial Condition will come from the restart file
+[ICs]
+  active = ''
+  [./u_var]
```

Comment:
Oh yeah - I missed the "active" line.  I would really prefer to remove the ICs block from restarting input files... it is just asking for problems.